tests/filecheck/dialects/stablehlo/test.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: xdsl-opt %s --parsing-diagnostics --verify-diagnostics --split-input-file | filecheck %s
+
+builtin.module {
+  func.func @different_shapes(%0: tensor<i32>) {
+    // CHECK: "stablehlo.abs"(%{{.*}}) : (tensor<i32>) -> tensor<i32>
+    %result = "stablehlo.abs"(%0) : (tensor<i32>) -> tensor<i32>
+    func.return
+  }
```

Comment:
Just as a little nit that will make life easier later, we try to make our tests as specific as possible, and for that we use the `test` dialect, which lets us just conjure up ssa values:
```suggestion
  %t0 = "test.op"() : () -> tensor<i32>

  // CHECK: "stablehlo.abs"(%{{.*}}) : (tensor<i32>) -> tensor<i32>
  %r0 = "stablehlo.abs"(%t0) : (tensor<i32>) -> tensor<i32>
```